infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLog.java

```diff
@@ -133,4 +133,11 @@ default OverwriteCause getOverwriteCauseForAddress(long address, LogData entry)
         }
         return cause;
     }
+
+    /**
+     * Query if the StreamLog has enough quota to accept writes
+     */
+    default boolean quotaExceeded() {
```

Comment:
why we need this method?

Reply:
I mean why it's `default`? What's the point for that

Reply:
For an in-memory StreamLog there is no quota. Will remove the in-memory implementation in the future.